core/templates/tuple.h

```diff
@@ -85,6 +85,10 @@ struct Tuple<T, Rest...> : Tuple<Rest...> {
 			value(std::forward<F>(f)) {}
 };
 
+// Tuple is zero-constructible if and only if all constrained types are zero-constructible.
+template <typename... Types>
+struct is_zero_constructible<Tuple<Types...>> : std::bool_constant<(is_zero_constructible_v<Types> && ...)> {};
```

Comment:
This may be simpler:
```suggestion
struct is_zero_constructible<Tuple<Types...>> 
    : std::conjunction<is_zero_constructible<Types>...> {};
```

Could also be applied to the other `bool_constant` uses.

Reply:
I *knew* there had to be an easier way to do that!